spec/statuspageApi.spec.js

```diff
@@ -123,8 +148,35 @@ describe('#hasMatchingHashtag', () => {
       statuspageApi.data = getMockData();
     });
 
-    it('should return truthy', () => {
+    it('should return falsy', () => {
       expect(statuspageApi.hasMatchingHashtag()).toBeFalsy();
     });
   });
 });
+
+describe('#scheduledMaintenances', () => {
+  describe('prioritizes whichever is the most recent incident', () => {
+    beforeEach(() => {
+      firstMaintenanceDate = '2020-07-20T09:11:40.438-04:00'
+      scheduledMaintenanceBody = '#scheduledmaintenance'
+      statuspageApi.data = getMockData();
+    });
+
+    it('should return the most recent alert, regardless of type', () => {
+      expect(statuspageApi.incidentName()).toEqual('FirstMaintenance');
+    })
+  })
+  describe('if two alerts occur at the same time, prioritize incidents', () => {
+    beforeEach(() => {
+      firstIncidentsDate = '2020-07-20T09:11:40.438-04:00'
+      firstMaintenanceDate = '2020-07-20T09:11:40.438-04:00'
+      scheduledMaintenanceBody = '#scheduledmaintenance'
+      statuspageApi.data = getMockData();
+    });
+
+    it('should return incidents instead of scheduled maintenance', ()=> {
+      expect(statuspageApi.incidentName()).toEqual('FirstIncident');
+    })
+  })
```

Comment:
So the describe with the `#` are for each method, so just move these tests into the `#lastIncident` block (remember to rename if you renamed the method itself), but you could also keep them in a separate describe like `describe("when scheduled maintenance present")`